app/views/booking_requests/customer.html.erb

```diff
@@ -0,0 +1,39 @@
+<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
```

Comment:
We will do. But for now I'm waiting for the text-only equivalent. I mentioned that in the commit.